src/librustc_resolve/resolve_imports.rs

```diff
@@ -42,9 +42,10 @@ use std::{mem, ptr};
 #[derive(Clone, Debug)]
 pub enum ImportDirectiveSubclass<'a> {
     SingleImport {
-        target: Ident,
         source: Ident,
```

Comment:
so... `source` is the way it's represented in the crate that defined it and `target` is the way it's defined in the crate that uses it?

Again, docs on these fields would be great.

Reply:
If we look at `use path as rename` as a symlink, then "source" is what the symlink refers to, and "target" is the symlink itself.
(Specifically, `source` and `target` `Ident`s in this structure correspond to `use prefix::source as target` in the import.)